src/OrchardCore/OrchardCore.Abstractions/Localization/CultureOptions.cs

```diff
@@ -0,0 +1,12 @@
+namespace OrchardCore.Localization;
+
+/// <summary>
+/// Represents a programmable options for localization culture.
+/// </summary>
+public class CultureOptions
+{
+    /// <summary>
+    /// Gets or sets whether to ignore the system culture settings or not.
```

Comment:
Link to the issue in the summary for context, otherwise it won't be apparent what we're talking about and why we have the option.

Same in `CultureScope`.

Reply:
We didn't add that frequently, may be we could add extra explanation in remarks

Reply:
That's also good, but an issue link lets people see the full context.